flink-table/flink-table-common/src/main/java/org/apache/flink/table/plan/stats/TableStats.java

```diff
@@ -69,4 +69,19 @@ public TableStats copy() {
 		return copy;
 	}
 
+	public TableStats merge(TableStats other) {
```

Comment:
add some comments, this function's behavior is "intersect" rather than "merge", we might need to explain why we merge it in this way

Reply:
also need test case

Reply:
OK, but I think `merge` name is better.